TLM/TMPE.API/Notifier.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+
+namespace TrafficManager.API
+{
+    public class OnModifiedEventArgs
+    {
+        /// <summary>
+        /// node/segment (citizen/vehicle in future) that has been modified.
+        /// </summary>
+        public InstanceID InstanceID;
+
+        /// <summary>
+        /// Reserved: the sender that caused the rule to be modified.
+        /// </summary>
+        public object Sender;
+
+        /// <summary>
+        /// Reserved: The rule that has been modified. object type is decided by sender.
+        /// </summary>
+        public object Data;
+    }
+
+    public static class Notifier
+    {
+        public static event Action EventLevelLaoded;
+
+        // TODO [issue #967]: notify TTL Start/Stop events
+        public static event Action<OnModifiedEventArgs> EventModified;
+
+        public static void OnLevelLoaded() => EventLevelLaoded?.Invoke();
+
+        public static void OnModified(OnModifiedEventArgs args)
```

Comment:
Why those methods are public? Are you sure you want to allow other mods to call them?

Reply:
they are in the API assembly but being called by the TrafficManger assembly.